### PR TITLE
with APIs

### PR DESCRIPTION
**Pull request by navya-shajan**
.>Now user can search a movie in the search-bar and result will be shown. 
.>the result can be list of movies with the alphabet/combination of alphabets contained in the search-bar , words or the full movie name.
.>After typing in the search-bar need to press 'Enter' to view the results.
.>the result will show only the searched movie posters with a mouse hover effect for now.
@golovnevs  
@NadezhdaBabenko
@keponk
@MrTim

### CODE REVIEW

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: That is a good idea to put this api key in a constant.

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: you can name these two url as: IMG_URL and MOVIES_URL for instance. But ofc you can leave them as they are, but then just make them consistent - at the moment one name is in capital letters and the other one in small.

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: spaces around `=`. You can install Prettier for VSCode and it will do this things automatically.

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: try to not leave commented code in the code which is donna be reviewed and then merged to master. If you want to have a comment put it above the function (in case you want to describe/clarify how function works)

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: here is a good example of a good comment in the code :)

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: use string literals: 
```js
const newUrl = `${url}&query=${value}`;
```

and you don't need this variable actually. You can do this operation directly in fetch:
```js
fetch(`${url}&query=${value}`)...
```

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: spaces around ` = `. And check the rest of the code.

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: spaces around `=>`

File: movieApp/asset/app.js
Comment (MrTim) [inline movieApp/asset/app.js]: and here spacing: `if () {}`

File: movieApp/index.html
Comment (MrTim) [inline movieApp/index.html]: assets folder meant to be used only for images, icons or other media content. Styles can be in styles folder, since you can have multiple css files for different pages.

File: movieApp/index.html
Comment (MrTim) [inline movieApp/index.html]: same goes here about asset folder. Js code you can move to js folder (for ex.)

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice stuff :+1: Left some comments, please fix them and push again.

File: movieApp/asset/app.js
Comment (navya-shajan) [reply movieApp/asset/app.js]: ok

File: movieApp/asset/app.js
Comment (navya-shajan) [reply movieApp/asset/app.js]: I have prettier installed. I don't known why its not doing things automatically.

**Comment by MrTim** [comment]
@navya-shajan why you merged this PR to master? Non of the teachers approved it, so you should not merge any open pr if you didn't get at least one approval.

**Comment by navya-shajan** [comment]
@MrTim I thought you already reviewed the code and mentioned few changes and further code review will not be there, so i  pushed changes and merged it. 
should i revert the changes?

**Comment by MrTim** [comment]
> Nice stuff +1 Left some comments, please fix them and push again.

That what I wrote) And if you can't see the approved check mark (green one) then this PR is not approved and you should continue communication in comments.

No you can leave it like that, I just wanted to check what was misleading in that case.

**Comment by navya-shajan** [comment]
sorry!! 
will be waiting to be approved by the teachers next time on-wards. :)
thanks

**Pull request merged**
